### PR TITLE
Update default values for batchnorm

### PR DESCRIPTION
**Pull request by yechengxi**
change default batch norm momentum and eps to match the pytorch implementation. The previous setting gave us a slow update speed.

### CODE REVIEW

**Comment by aterzis-google** [comment]
Thanks for the PR Chengxi!

It's not one of Objax's goals to mirror PyTorch so it's OK to have different defaults. Also we would like to see more evidence that the new default values are better across a wide variety of settings.
